### PR TITLE
Save total, unbinned charge in sns_response table

### PR DESCRIPTION
**Pull request by paolafer**
This PR modifies the code to save the total charge detected by each sensor, instead of a waveform with charge in time bins.
The reason is that we never used that charge binned, we always sum the charge of each sensor before doing anything else, so there's no point in waisting space to save the time.
This change implies that the `time_binning` parameter now refers to the binning of the TOF sensors (5 ps) and is left as a parameter, because it could be used in principle to save the full waveform in a broader binning for studies of the electronic response.

### CODE REVIEW

File: source/sensdet/ToFSD.cc [hunk 1]
```diff
@@ -109,13 +109,13 @@ G4bool ToFSD::ProcessHits(G4Step *step, G4TouchableHistory *)
       {
         hit = new SensorHit();
         hit->SetPmtID(pmt_id);
-        hit->SetBinSize(timebinning_);
+        hit->SetBinSize(1.*millisecond);
```
Comment (carmenromo) [inline source/sensdet/ToFSD.cc:112]: What does this bin size refer to?

**Review by carmenromo — COMMENTED** [review]
When compiling appears a warning saying: `unused variable 'binsize' in line 324` in `source/persistency/PetaloPersistencyManager.cc`

File: source/sensdet/ToFSD.cc (hunk 1 above)
Comment (paolafer) [reply source/sensdet/ToFSD.cc]: I've eliminated this setting, since it's irrelevant.

**Comment by paolafer** [comment]
> When compiling appears a warning saying: `unused variable 'binsize' in line 324` in `source/persistency/PetaloPersistencyManager.cc`

I have eliminated the variable.

**Review by carmenromo — APPROVED** [review]
The `sns_response` output table has been modified by removing the `time_bin` column since it didn't provide additional information. Aproved!

**Pull request merged**